src/components/NavBar.vue

```diff
@@ -7,7 +7,7 @@
       <v-spacer></v-spacer>
       <v-btn outlined id="menu" v-if="!loggedIn" to="Login">Iniciar Sesión</v-btn>
       <v-btn outlined id="menu" v-if="!loggedIn" to="Register">Registrarse</v-btn>
```

Comment:
Deberías reemplazar `to="Register"` por `to="/register"
`
Por los motivos antes mencionados